index.html

```diff
@@ -0,0 +1,106 @@
+          NFT BLOCKCHAIN
+          GLOBAL EVENT
+          2022
+        </h1>
+        <p class="boxWhite">All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet. It uses a dictionary of over 200 Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable.</p>
+        <h2 class="h2Main coco">2022.09.25 - 10.12</h2>
+        <h3 class="marginTop bold">Event Partner Webex. Join Online</h3>
+      </section>
+      <section class="program">
+        <h2 class="bold">Event Features</h2>
+        <p class="line"></p>
+        <div class="programCard sideMargin">
+          <div class="item">
+            <img src="img/icon1.svg" alt="Webinars">
+            <h3 class="bold red">Webinars</h3>
+            <p>Morbi non vulputate magna. Duis iaculis purus quis dolor varius ullamcorper.</p>
+          </div>
+          <div class="item">
+            <img src="img/icon4.svg" alt="Networking">
+            <h3 class="bold red">Networking</h3>
+            <p>Morbi non vulputate magna. Duis iaculis purus quis dolor varius ullamcorper.</p>
+          </div>
+          <div class="item">
+            <img src="img/icon2.svg" alt="Scaling">
+            <h3 class="red bold">Scaling</h3>
+            <p>Morbi non vulputate magna. Duis iaculis purus quis dolor varius ullamcorper.</p>
+          </div>
+          <div class="item">
+            <img src="img/icon3.svg" alt="Integration">
+            <h3 class="bold red">Integration</h3>
+            <p>Morbi non vulputate magna. Duis iaculis purus quis dolor varius ullamcorper.</p>
+          </div>
+        </div>
+        <button id="join" class="fontMid radius" type="button" name="join">Join NFT Global Event</button>
+      </section>
+
+      <section id="participants">
+        <h2 class="alignCenter bold">Participants</h2>
+        <p class="line"></p>
+        <div class="gridParticipants">
+        </div>
+        <button id="more" class="fontMid radius btn" type="button" name="more">See More</button>
+        <button id="less" class="fontMid radius btn" type="button" name="less">See Less</button>
+      </section>
+
+      <section class="partner" id="partners">
+        <h2 class="marginTopBottom bold">Partners</h2>
+        <p class="line"></p>
+        <div class="partnerWrapper sideMargin">
+          <img src="img/cb.png" alt="Coinbase logo">
+          <img src="img/cc.png" alt="Coinmarketcap logo">
+          <img src="img/forbes.png" alt="Forbes logo">
+          <img src="img/cardano.png" alt="Cardano logo">
+          <img src="img/ripple.png" alt="Ripple logo">
+        </div>
+      </section>
+      <footer class="wrapper">
+        <div id="footer" class="">
+          <img src="img/logo.svg" alt="NFT GE Logo">
+            <p>NFT Global Event USA
+            <br>Some Rights Reserved</p>
+        </div>
+      </footer>
```

Comment:
- [ ] Please make sure that the partners' section and the footer are visible in the mobile size of the home page as it is visible on the about page.
![Screenshot from 2022-05-21 16-57-55](https://user-images.githubusercontent.com/33752735/169657276-0e4f5eff-bd71-42fb-b02b-ba1131f88f4c.png)